src/main/kotlin/no/nav/tiltakspenger/arena/repository/ArenaSakDTO.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.tiltakspenger.arena.repository
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class ArenaSakDTO(
+    val aar: Int,
+    val lopenrSak: Long,
+    val status: ArenaSakStatus,
+    val ytelsestype: ArenaYtelse,
+    val ihtVedtak: List<ArenaVedtakDTO>,
+) {
+
+    /*
+    TODO: Blir det det samme? Må kanskje filtrere litt mer..
+
+          (SELECT to_char(min(nvl(va.fra_dato,va.reg_dato)),'rrrr-mm-dd')||'T00:00:00'
+          FROM vedtak va
+              ,rettighettype rht
+          WHERE va.sak_id=sak.sak_id
+          AND va.rettighetkode = rht.rettighetkode
+          AND va.vedtaktypekode NOT IN ('S')
+          AND rht.sakskode IN ('AA','INDIV')
+          AND rht.rettighetkode NOT IN ('AA115')
+          AND va.utfallkode != 'AVBRUTT'
+         ) AS "fomGyldighetsperiode"
+
+         (SELECT DISTINCT first_value(to_char(va.til_dato,'rrrr-mm-dd')||'T00:00:00')
+                   OVER (ORDER BY fra_dato DESC, til_dato DESC)
+          FROM vedtak va
+          WHERE va.sak_id=sak.sak_id
+          AND va.vedtaktypekode IN ('O','E','G')
+         ) AS "tomGyldighetsperiode"
+     */
```

Comment:
Hva er dette for noe? "Blir det det samme som".. hva da? :)

Reply:
Det er en kommentar knyttet til kodenlinjen under. Som jeg mistenker er litt for simpel. Tanken min var å deploye det for så å kunne se på diffen mellom outputen fra webservicen og outputen fra db for å se om jeg har gjort det godt nok. Så denne kan vi komme litt tilbake til?